bin/prometheus-mock-data.sh

```diff
@@ -21,7 +21,7 @@ run_container() {
 entrypoint() {
   CLUSTER_ID=${CLUSTER_ID:-test01}
   BILLING_PROVIDER=${BILLING_PROVIDER:-aws}
-  METRICS="${METRICS:-kafka_id:kafka_broker_quota_totalstorageusedbytes:max_over_time1h_gibibytes}"
+  METRICS="${METRICS:-kafka_id:kafka_broker_quota_totalstorageusedbytes:max_over_time1h_gibibyte_months}"
```

Comment:
Do we need to change max_over_time1h_gibibyte_months in test_tag_profile.yaml?

Reply:
Not needed; this was only present in RHM for the sake of handling storage as measured per-hour.